ziti/cmd/ziti/cmd/edge_controller/create_service.go

```diff
@@ -33,7 +33,7 @@ type createServiceOptions struct {
 	roleAttributes     []string
 	configs            []string
 	requireEncryption  bool
-	optionalEncryption bool
+	encryption         encryptionVar
```

Comment:
shouldn't the changelog get updated too?

Reply:
I second this.

Reply:
done